libs/sdk-ui-tests/stories/visual-regression/kit/ShareDialog/ShareGranteeBase.tsx

```diff
@@ -2,35 +2,58 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { InternalIntlWrapper } from "@gooddata/sdk-ui-ext/dist/internal/utils/internalIntlProvider";
+import { uriRef } from "@gooddata/sdk-model";
+import { action } from "@storybook/addon-actions";
+import { GranteeItem, IGranteeUser, IGranteeInactiveOwner, ShareGranteeBase } from "@gooddata/sdk-ui-kit";
+
 import { UiKit } from "../../../_infra/storyGroups";
 import { withScreenshot } from "../../../_infra/backstopWrapper";
 import { wrapWithTheme } from "../../themeWrapper";
 
-import "@gooddata/sdk-ui-kit/styles/css/main.css";
-import { action } from "@storybook/addon-actions";
-
-import { GranteeItem, IGranteeUser, IGranteeInactiveOwner, ShareGranteeBase } from "@gooddata/sdk-ui-kit";
 import { grantees, inactiveUser, owner } from "./GranteeMock";
-import { uriRef } from "@gooddata/sdk-model";
+
+import "@gooddata/sdk-ui-kit/styles/css/main.css";
 
 interface BasicExampleProps {
     isDirty: boolean;
     grantees: GranteeItem[];
     owner: IGranteeUser | IGranteeInactiveOwner;
+    isUnderStrictControl?: boolean;
+    isLocked?: boolean;
+    isLockingSupported?: boolean;
+    isUnderStrictControlSupported?: boolean;
 }
 
-const BasicExample = (props: BasicExampleProps): JSX.Element => {
+const BasicExample: React.FC<BasicExampleProps> = ({
+    owner,
+    grantees,
+    isDirty,
+    isLocked = false,
+    isUnderStrictControl = true,
+    isLockingSupported = true,
+    isUnderStrictControlSupported = true,
+}) => {
     return (
         <div id="Share-Grantee-base-basic-example">
             <ShareGranteeBase
-                isDirty={props.isDirty}
+                isDirty={isDirty}
                 isLoading={false}
-                owner={props.owner}
-                grantees={props.grantees}
+                owner={owner}
+                grantees={grantees}
                 onGranteeDelete={action("onGranteeDelete")}
                 onAddGranteeButtonClick={action("onAddGrantee")}
                 onCancel={action("onCancel")}
                 onSubmit={action("onSubmit")}
+                isUnderStrictControl={isUnderStrictControl}
+                isLocked={isLocked}
+                isLockingSupported={isLockingSupported}
+                isUnderStrictControlSupported={isUnderStrictControlSupported}
+                onStrictControlChange={action("onStrictControlChange")}
+                onLockChange={action("onLockChange")}
+                labels={{
```

Comment:
Could you add this labels into sep. file and share them cross all stories.

Reply:
Done.